src/Provider/TwigServiceProvider.php

```diff
@@ -10,26 +10,44 @@
 use Bolt\Twig\SafeEnvironment;
 use Bolt\Twig\SecurityPolicy;
 use Silex\Application;
-use Silex\ServiceProviderInterface;
+use Pimple\ServiceProviderInterface;
 use Symfony\Bridge\Twig\AppVariable;
 use Symfony\Bridge\Twig\Extension\AssetExtension;
 use Symfony\Bridge\Twig\Extension\HttpFoundationExtension;
+use Pimple\Container;
 
 class TwigServiceProvider implements ServiceProviderInterface
 {
-    public function register(Application $app)
+    public function register(Container $app)
     {
+        // Twig options
+        $app['twig.options'] = function () use ($app) {
```

Comment:
Pimple does not let you override frozen services, and after the TwigServiceProvider is registered, these options seem to be frozen. Moved before we instantiate the TwigServiceProvider below, however that suggests that it might be registerred before.